src/akamai/js/akamaiHD_flash.js

```diff
@@ -37,6 +39,7 @@ require("../../../html5-common/js/utils/constants.js");
    * @class OoyalaAkamaiHDFlashVideoFactory
    * @classdesc Factory for creating video player objects that use Flash in an HTML5 wrapper.
    * @property {string} name The name of the plugin
+   * @property {boolean} ready The readiness of the plugin for use.  True if elements can be created.
```

Comment:
This was removed.  Based on this and some of the additions below I am wondering if this file is re-adding items which were previously removed